src/main/webapp/app/overview/course-dashboard/course-dashboard.component.ts

```diff
@@ -52,11 +54,15 @@ export class CourseDashboardComponent implements OnInit, OnDestroy {
         private route: ActivatedRoute,
         private router: Router,
         private courseDashboardService: CourseDashboardService,
+        private irisSettingsService: IrisSettingsService,
     ) {}
 
     ngOnInit(): void {
         this.paramSubscription = this.route.parent?.parent?.params.subscribe((params) => {
             this.courseId = parseInt(params['courseId'], 10);
+            this.irisSettingsService.getCombinedCourseSettings(this.courseId).subscribe((settings) => {
```

Comment:
I'm getting an error
<img width="1027" alt="image" src="https://github.com/ls1intum/Artemis/assets/5898705/b4872d04-b958-4670-90e0-2e0de95d6d67">

Reply:
With "activated" you mean the Spring profile, right? Otherwise the 405 would make no sense

Reply:
Added a profile check